src/main/java/io/ebeaninternal/server/deploy/BeanDescriptor.java

```diff
@@ -2153,7 +2156,7 @@ protected ElPropertyValue buildElGetValue(String propName, ElPropertyChainBuilde
 
     if (propertyDeploy && chain != null) {
       ElPropertyDeploy fk = elDeployCache.get(propName);
-      if (fk != null && fk instanceof BeanFkeyProperty) {
+      if (fk instanceof BeanFkeyProperty) { // instanceof will return false if the variable is null, so a not-null check is redundant.
```

Comment:
As the comment says: instanceof a null object will always return false. So an extra null check is not necessary.